### PR TITLE
View Online Applicants UI

### PR DESCRIPTION
**Pull request by syriazapanta**

### CODE REVIEW

File: application/modules/hirer/views/viewapplication.php [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                                echo sprintf(
+                                    lang('label_closing_date'),
+                                    date_display($job['closing_date'])
+                                );
+                                ?></span></li><br/>
+                    </ul>
+                    <hr>
+                </div>
+                <p class="badge badge-light"><?php echo sprintf(lang('label_no_view'), $jobView); ?></p>
+                <a href="#" class="badge badge-info float-right"><i class="simple-icon-eye align-middle"></i> View Sample of Ad</a>
```
Comment (princeryansy) [inline application/modules/hirer/views/viewapplication.php]: please add raw text in lang.

File: application/modules/hirer/views/viewapplication.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                                        ?>
+                                        <?php $data = array(
+                                            $checkbox,
+                                            '<div class="row">
+                                                        <div class="col">
+                                                            <ul class="list-unstyled my-auto">
+                                                                <li><a href="' . url_hirer('view-resume/' . $row['applicantId']) . '" class="font-weight-bold" target="_blank">' . $row['applicantName'] . '</a></li>
+                                                                <li class="text-muted"><small><em>Last updated: <strong>'. date_display($row['lastUpdated']) . '</strong></em></small></li>
+                                                                <hr class="my-2">
+                                                                <li><small>Resume:</small> <span class="badge badge-info">Unread</li>
```
Comment (princeryansy) [inline application/modules/hirer/views/viewapplication.php]: please add raw texts in lang

File: application/modules/hirer/views/viewapplication.php [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                                                                    <li><small>Availability: <strong>' . $row['availability']. '</strong></small></li>
```
Comment (princeryansy) [inline application/modules/hirer/views/viewapplication.php]: please add raw texts in lang

File: assets/js/hirer/viewapplications.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+$(document).ready(function (e) {
+    $(".select2-clear").select2({
+      theme: "bootstrap",
+      placeholder: "Move applicants to folder",
```
Comment (princeryansy) [inline assets/js/hirer/viewapplications.js]: please add raw texts in lang

**Comment by syriazapanta** [comment]
@princeryansy Please see updated changes. Thank you!

**Pull request merged**